frontend/src/common/models/user/IUserState.ts

```diff
@@ -0,0 +1,9 @@
+import { IUser } from './IUser';
+
+interface IUserState {
+  data: IUser | null;
```

Comment:
```suggestion
  user?: IUser;
```

interface for state should be in reducer